app/jobs/clock/terminate_ended_subscriptions_job.rb

```diff
@@ -8,8 +8,8 @@ def perform
       Subscription
         .joins(customer: :organization)
         .active
-        .where("DATE(subscriptions.ending_at#{Utils::TimezoneService.at_time_zone}) = "\
-               "DATE(?#{Utils::TimezoneService.at_time_zone})", Time.current)
+        .where("DATE(subscriptions.ending_at#{Utils::Timezone.at_time_zone}) = "\
+               "DATE(?#{Utils::Timezone.at_time_zone})", Time.current)
```

Comment:
Layout/MultilineMethodArgumentLineBreaks: Each argument in a multi-line method call must start on a separate line.